aws-copilot.hcl

```diff
@@ -0,0 +1,48 @@
+description = "AWS Copilot is an open source command line interface that makes it easy for developers to build, release, and operate production ready containerized applications on AWS App Runner, Amazon ECS, and AWS Fargate."
+test = "copilot --help"
+binaries = ["copilot"]
+
+darwin {
```

Comment:
🤔 for these two, I'm curious if I could DRY it up a bit...

```diff
binaries = ["copilot"]
+ source = "https://github.com/aws/copilot-cli/releases/download/v${version}/copilot-${os}"
+ on unpack {
+    rename { from = "${root}/copilot-${os}" to = "${root}/copilog" }
+ }

- darwin {
```

I was hesitant to do this because I didn't know if it would attempt to do this for every OS and break Windows and Linux arch's.

Reply:
I had a look at their releases page and have more suggestions below. I think you can DRY this a fair bit more.